.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Not needed for everyone

Reply:
Thanks proful. Will delete these files next time before raising a PR